experiments/react-native-uifabric/src/components/RadioGroup/RadioGroup.settings.ts

```diff
@@ -0,0 +1,26 @@
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { IRadioGroupType, radioGroupName } from './RadioGroup.types';
+import { ViewProps } from 'react-native';
+
+export const settings: IComposeSettings<IRadioGroupType> = [
+  {
+    root: {
+      accessible: true,
+      accessibilityRole: 'radio',
+      style: {
+        display: 'flex',
+        alignItems: 'flex-start',
+        flexDirection: 'column',
+        alignSelf: 'flex-start'
```

Comment:
why is `alignSelf` needed?

Reply:
Took this from Button.settings.ts, but it looks like it's not necessary here. Removing it